projects/go-lib/package.json

```diff
@@ -9,7 +9,7 @@
   "description": "A library of UI components for Angular v7+.",
   "license": "MIT",
   "peerDependencies": {
-    "@ng-select/ng-select": "^2.20.5"
+    "@ng-select/ng-select": "~3.7.3"
   },
   "devDependencies": {
     "@angular/common": "^7.0.0",
```

Comment:
Do we need to update these dependencies to be angular 8?

Reply:
We should give a range, as I think we are compatible with multiple versions.

Reply:
oh woops, these are dev dependencies, not peer. Yeah it should be 8.

Reply:
@eligolding #632